### DISCUSSION TITLE
antd 重复打包

### DISCUSSION THREAD

**Discussion by markstock7** [umijs/umi#6021] [Q&A]
使用 umi3, 并且配置了 splitChunks

`
      name: 'antd',
      test: /antd[\\/]es|antd|@ant-design|node_modules[\\/]rc-(.*)/,
    
`

但是从打包的代码中看看， vendors-antd 中有一些 antd 的代码，umi.js 中也包含了大量 antd 的代码

请问这个是什么原因导致的？

**Comment by xiaohuoni** [comment]
antd 是按需加载的，到这里匹配的名字是 antd/button 之类的名字，而不是 antd。
